src/three/plugins/ImplicitTilingPlugin.js

```diff
@@ -0,0 +1,63 @@
+import {SUBTREELoader} from "./SUBTREELoader.js";
+
+export class ImplicitTilingPlugin {
+
+	constructor() {
+
+		this.name = 'IMPLICIT_TILING_PLUGIN';
+
+	}
+
+	init(tiles) {
+
+		this.tiles = tiles;
+
+	}
+
+	preprocessNode(tile, uri, parentTile) {
+
+		if (tile.implicitTiling) {	//only for root of the tileset
+
+			tile.__hasUnrenderableContent = true;
+			tile.__hasRenderableContent = false;
+
+			// Store the infos from the tileset
+			tile.__subtreeDivider =  tile.implicitTiling.subdivisionScheme === "QUADTREE" ? 4 : 8;
+			tile.__subtreeUri = tile.implicitTiling.subtrees.uri;
+
+			// Original content uri
+			tile.__contentUri = uri ?? tile.content?.uri;
+
+			// Declare some properties
+			tile.__subtreeIdx = 0;	// Idx of the tile in its subtree
+
+			// Coords of the tile
+			tile.__x = 0;
+			tile.__y = 0;
+			tile.__z = 0;
+
+			let implicitUri = tile.__subtreeUri.replace("{level}", (tile.__depth ?? tile.__level) ?? 0);
+			implicitUri = implicitUri.replace("{x}", "0");
+			implicitUri = implicitUri.replace("{y}", "0");
+			implicitUri = implicitUri.replace("{z}", "0");
+			tile.content.uri = new URL(implicitUri, tile.__basePath + '/').toString();
```

Comment:
Is it necessary to assign this? The tile is marked as unrenderable so the content uri will never be used. Or is it the case that this tile _is_ renderable?

Reply:
In this line, we replace the content uri:
```js
  "content": {
      "uri": "content/{level}_{x}_{y}.glb"
    },

```
by 
```js
  "content": {
      "uri": "content/0_0_0.subtree"
    },

```
This will be the root subtree, so it needs to be fetch and parse, even though the file .subtree itself doesn't contains 3D object but only information on content availability.

These flags
``` js
tile.__hasUnrenderableContent = true;
tile.__hasRenderableContent = false;
```
seem to still allow the fetch and parse. Is that a bug ?
What''s the meaning of the 2 RenderableContent flags ? Renderable doesn't mean 3D data to be render on the screen ?
Am I not using it correctly ?

Reply:
> In this line, we replace the content uri:
> ...
>      "uri": "content/{level}_{x}_{y}.glb"
> ...
> by
> ...
>      "uri": "content/0_0_0.subtree"

OH. Sorry I missed that the original glb extension was replaced with the subtree uri.

> What''s the meaning of the 2 RenderableContent flags ? Renderable doesn't mean 3D data to be render on the screen ?

There are three content flags:

- `__hasContent` mean that the tile has content of any kind that has to be loaded - whether that be 3d geometry or data that should be expanded to child tiles.
- `__hasUnrenderableContent` means that the tile has content like an external tile set but has no geometry to render.
- `__hasRenderableContent` means the tile has content that is renderable like 3d geometry or points.

Given the above extension mixup, you're using this correctly, sorry about that.